app/script/conversation/ConversationRepository.coffee

```diff
@@ -1242,13 +1255,29 @@ class z.conversation.ConversationRepository
       amplify.publish z.event.WebApp.ANALYTICS.EVENT, z.tracking.EventName.FILE.UPLOAD_FAILED, tracking_data
 
   ###
-  Delete message in conversation.
+  Delete message for everyone.
 
-  @param conversation_et [z.entity.Conversation] Conversation to post the file
-  @param message_et [z.entity.Message] Message object
+  @param conversation_et [z.entity.Conversation]
+  @param message_et [z.entity.Message]
   ###
-  delete_message: (message_et) =>
-    conversation_et = @active_conversation()
+  delete_message_everyone: (conversation_et, message_et) =>
+    if message_et?
```

Comment:
We are calling this via amplify. Safeguard against an undefined `conversation_et` as well?

`if conversation_et and message_et`